frontend/src/components/body/hooks/useFetchFriends.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect } from "react";
+
+import { User } from "../types/user";
+
+type UserId = User["id"];
+
+const useFetchFriends = (
+  api: string,
+  userId: UserId,
+  setUserList: React.Dispatch<React.SetStateAction<User[]>>
```

Comment:
should this be setUserList, or onSuccess?

Reply:
lets discuss naming today